rest/data/host_test.go

```diff
@@ -209,6 +211,10 @@ func (s *HostConnectorSuite) TestSpawnHost() {
 	const testUserID = "TestSpawnHostUser"
 	const testUserAPIKey = "testApiKey"
 	const testInstanceType = "testInstanceType"
+	config, err := evergreen.GetConfig()
+	s.NoError(err)
+	config.SpawnHostsPerUser = cloud.DefaultMaxSpawnHostsPerUser
+	s.NoError(config.Set())
```

Comment:
@john-m-liu having to set this here doesn't indicate an error, right? In prod the config will be set from the UI before errors could occur?

Reply:
Correct. It might be cleaner to set in the setup for the suite, but it's not a problem